src/services/Omnibridge/Socket/SocketBridge.ts

```diff
@@ -318,14 +318,16 @@ export class SocketBridge extends OmnibridgeChildBase {
 
       const value = parseUnits(from.value, from.decimals)
 
+      if (!from.chainId || !to.chainId || !this._account) return
```

Comment:
We need to try-catch fetching with API. On error (404,500) it will early exit the code.